docs/shared/react-standalone-tutorial/react-standalone.md

```diff
@@ -834,7 +834,7 @@ Here's some more things you can dive into next:
 
 Also, make sure you
 
-- [Join the Nx community Slack](https://go.nrwl.io/join-slack) to ask questions and find out the latest news about Nx.
+- [Join the Official Nx Discord Server](http://go.nx.dev/community) to ask questions and find out the latest news about Nx.
```

Comment:
Missed one.

Could you open a separate PR to change all the `http://` in here please?